### PR TITLE
ENG 92: Implement workspace settings

### PR DESCRIPTION
**Pull request by afolasope**

### CODE REVIEW

**Comment by jdamilola** [comment]
@afolasope What's the update on this PR?

File: apps/frontend/src/modules/workspace/components/settings/Workspace/components/MemberList.tsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            );
+            setFilteredMembers(filtered as OrganizationMemberCountableEdge[]);
+        } else {
+            setFilteredMembers(workspace?.members?.edges as OrganizationMemberCountableEdge[]);
+        }
+    }, [searchValue, workspace?.members?.edges]);
+
+    const handleLeaveOrganization = async (organizationId: string) => {
+        const response = await leaveOrganization(organizationId);
+        console.log(organizationId, response);
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/components/settings/Workspace/components/MemberList.tsx:42]: @afolasope This too. Also, check all occurrences of `console.log`, then remove them.

File: apps/frontend/src/modules/auth/AuthProvider.tsx [hunk 2]
```diff
@@ -352,21 +368,22 @@ export const AuthProvider = ({ children, apollo }: AuthProviderProps) => {
...
-                        ({ node }) => node.id === user.project?.id,
-                    )?.node;
-                    updateUserCache({
-                        ...data.viewer,
-                        organization: convertToAuthOrganization(organization),
-                        project,
-                    });
-                }
+                // if (data?.viewer) {
+                //     const organization = data.viewer.organizations?.edges.find(
```
Comment (jdamilola) [inline apps/frontend/src/modules/auth/AuthProvider.tsx:374]: @afolasope Why did you remove this? If it's no longer needed, please remove them instead of commenting them out.

File: apps/frontend/src/modules/workspace/components/settings/Workspace/components/MemberList.tsx (hunk 1 above)
Comment (afolasope) [reply apps/frontend/src/modules/workspace/components/settings/Workspace/components/MemberList.tsx]: I have removed them

File: apps/frontend/src/modules/auth/AuthProvider.tsx (hunk 2 above)
Comment (afolasope) [reply apps/frontend/src/modules/auth/AuthProvider.tsx]: I have attended to this.

**Review by jdamilola — APPROVED** [review]
